content/1.7/back-office/international/locations/zones/zones-listing.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Zones listing
+weight: 1
+---
+
+# Zones listing
+
+## Call to action
+
+A call to actions is displayed in the header of this page:
+
+ - Add new zone
+
+👉  See the [specs of the add new zone]({{<ref "add-new-zone.md">}} "Add new zone") 
+
+## Settings wheel
+
+When clicking on the settings wheel, the following options are displayed:
+
+1. Refresh list
+
+After clicking on it, the page is reloaded. Sorts and searches are kept.
+
+2. Show SQL query
+
+After clicking on it, the SQL query is displayed in a modal. The SQL query takes into account filters and sorts that have been done.
+
+3. Export to SQL Manager
+
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+# Bulk actions
+
+Clicking on the _Bulk actions_ button allows performing bulk actions for the selected zones (using the checkboxes). A user can:
+
+- Enable/Disable selection: these buttons will enable/disable all the checkboxes at the beginning of each row of the listing.
+- Delete selected: This button will delete the selected zone. 
+
+## Columns
+
+
+The page displays the zones list with the following columns:
+
+1. ID
+
+It's possible to sort by id asc or desc and search by id.
+
+2. Zone
+
+It's possible to search by a zone.
+
+3. Enabled
+
+A drop-down allows to select: 
+- Yes
+- No
+When a search is performed with "yes" selected, only the zones that open in a new window are displayed.
```

Comment:
I don't understand why you talk about a new window ?

Reply:
Not sure why I wrote new window, same one, just new view of only the searched zones:D